src/patient-registration/patient-registration.component.tsx

```diff
@@ -463,6 +463,7 @@ export const PatientRegistration: React.FC = () => {
     // handle save patient
     savePatient(abortController, patient, patientUuidMap['patientUuid'])
       .then(response => {
+        // TODO this entire block of code should be migrated somewhere...
```

Comment:
Looks all good to me.

I think the intention here is sound. Can we just do it ;)?

Reply:
i can do it - just wondering if this block should be migrated into like a services class or util module. haha, i didn't want to jump the gun yet just in case i would break something... 😅

Reply:
oof, actually, there's like 100 lines of code just stashed in one onSubmit function... this will take a bit to refactor - maybe another day or two. I will create a follow up for it as another draft pr...